Python/Setup/PythonToolsInstaller/PythonToolsInstaller.wixproj

```diff
@@ -77,10 +76,6 @@
       <Name>UwpMsm</Name>
       <Project>{83946E81-7A1B-4D3B-927F-4CD67AC95BE7}</Project>
     </ProjectReference>
-    <ProjectReference Include="$(PTVSPRPath)VsLoggerSetup\VsLogger.wixproj" Condition="'$(IncludeVsLogger)' == 'True'">
```

Comment:
It's using the wrong system, so we're not getting any data from it anyway. When we figure out how to use the new one then we'll add that in.